datafusion-cli/src/helper.rs

```diff
@@ -197,6 +204,19 @@ pub fn unescape_input(input: &str) -> datafusion::error::Result<String> {
     Ok(result)
 }
 
+/// Splits a string which consists of multiple queries.
+pub(crate) fn split_from_semicolon(sql: String) -> Vec<String> {
+    sql.split(';')
```

Comment:
We may want to add a test? Just wondering if it will work for query 
```
select ';'
```
?

Reply:
I will send a commit excluding semicolons if they are in "_" or '_', and some tests.